include/fast_methods/ndgridmap/ndgridmap.hpp

```diff
@@ -35,6 +35,7 @@
 #include <cstddef>
 #include <array>
 #include <sstream>
+#include <cmath>
```

Comment:
I just copy the wrong file name. I added the **cmath** library to properly include std::isinf, else wise it cannot find it.